src/pkg/packaging/snaps/dotnet-sdk-3/dotnet-runtime

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+dotnet_runtime_snaps=$(ls $SNAP/../..|grep dotnet-runtime)
+
+for snap in $dotnet_runtime_snaps
+do
+  runtime_dir=$(realpath $SNAP/../../$snap/current/shared/Microsoft.NETCore.App)
```

Comment:
Hadn't seen `realpath` before (vs. `readlink -f`) but the difference is too complicated for me to quickly get to an opinion. 😄 

Missing quotes though:

```suggestion
  runtime_dir=$(realpath "$SNAP/../../$snap/current/shared/Microsoft.NETCore.App")
```

Reply:
This script was written by the snapcraft dev lead so I wasn't in a position to question ;-)